src/Numerics/DGMethods/DGModel_kernels.jl

```diff
@@ -928,6 +1102,427 @@ fluxes, respectively.
     end
 end
 
+@doc """
+    function vert_fvm_interface_tendency!(
+        balance_law::BalanceLaw,
+        ::Val{info},
+        ::Val{nvertelem},
+        ::Val{periodicstack},
+        ::VerticalDirection,
+        numerical_flux_first_order,
+        tendency,
+        state_prognostic,
+        state_auxiliary,
+        vgeo,
+        sgeo,
+        t,
+        vmap⁻,
+        vmap⁺,
+        elemtobndy,
+        elems,
+        α,
+    )
+
+Compute kernel for evaluating the interface tendencies using vertical FVM
+reconstructions with a DG method of the form:
+
+∫ₑ ψ⋅ ∂q/∂t dx - ∫ₑ ∇ψ⋅(Fⁱⁿᵛ + Fᵛⁱˢᶜ) dx + ∮ₑ n̂ ψ⋅(Fⁱⁿᵛ⋆ + Fᵛⁱˢᶜ⋆) dS,
+
+or equivalently in matrix form:
+
+dQ/dt = M⁻¹(MS + DᵀM(Fⁱⁿᵛ + Fᵛⁱˢᶜ) + ∑ᶠ LᵀMf(Fⁱⁿᵛ⋆ + Fᵛⁱˢᶜ⋆)).
+
+This kernel computes the surface terms: M⁻¹ ∑ᶠ LᵀMf(Fⁱⁿᵛ⋆ + Fᵛⁱˢᶜ⋆)), where M
+is the mass matrix, Mf is the face mass matrix, L is an interpolator from
+volume to face, and Fⁱⁿᵛ⋆, Fᵛⁱˢᶜ⋆ are the numerical fluxes for the inviscid
+and viscous fluxes, respectively.
```

Comment:
Awesome doc string. This is super helpful, very clear, and helps connect the dots 😃

Reply:
I agree it is great! I have one question though: 

In the discrete equation there is the term `M⁻¹MS`, which I believe it represents the source term. If I am not blind at the moment (which I actually am partially ^_^") I do not see it in the continuous equation though